website/tailwind.config.mjs

```diff
@@ -1,11 +1,40 @@
 import { addIconSelectors } from '@iconify/tailwind';
 import daisyui from 'daisyui';
 
+const colors = {
+    indigo: '#473894', // rgba(51, 34, 136, 0.9)
+    green: '#117733',
+    cyan: '#35a6de', // I modified '#88CCEE' manually to make it darker
+    teal: '#44AA99',
+    olive: '#999933',
+    sand: '#DDCC77',
```

Comment:
I played around with this a bit and found `sand: '#BBAA55'` slightly better, because it increases the contrast:

Old:
![grafik](https://github.com/user-attachments/assets/e886bfa2-9727-4f74-8094-99ffbe048ecf)

New:
![grafik](https://github.com/user-attachments/assets/cb87a5eb-0577-49a7-b430-0b423dd83669)